src/dotnet/Host/AppHost.cs

```diff
@@ -106,6 +106,8 @@ protected virtual void ConfigureAppConfiguration(IConfigurationBuilder appBuilde
             appBuilder.AddJsonFile(item.Path, item.Optional, reloadOnChange: false);
         }
         appBuilder.AddJsonFile("appsettings.local.json", optional: true, reloadOnChange: false);
+        appBuilder.AddEnvironmentVariables();
```

Comment:
![image](https://user-images.githubusercontent.com/1673423/163073989-02240a7a-b540-48fd-b7b5-a55ae7c0ca5d.png)

Reply:
I am 99.99% sure it's already there, because we use env. vars locally & in docker-compose.

Reply:
yes, but the order is incorrect. Environment variables must be registered the last to have the highest priority.